lib/prefix.h

```diff
@@ -321,6 +321,15 @@ union prefixconstptr {
 /* Maximum string length of the result of prefix2str */
 #define PREFIX_STRLEN 80
 
+/*
+ * Longest possible length of a (S,G) string is 36 bytes
+ * 123.123.123.123 = 16 * 2
```

Comment:
Isn't this 15*2?   Max string should be 34 then.

Reply:
this was old code moved in from pim to lib. however 34 does seem right ipaddr string length is 15 not 16. I will fixup in a separate commit and highlight this.

Reply:
tagging @donaldsharp  so he knows this change is happening